lib/login_screen.dart

```diff
@@ -1,151 +1,240 @@
+                              enabledBorder: OutlineInputBorder(
+                                  borderSide:
+                                      BorderSide(color: textField_fill_color),
+                                  borderRadius: BorderRadius.circular(12.0)),
+                              focusedBorder: OutlineInputBorder(
+                                  borderSide:
+                                      BorderSide(color: textField_fill_color),
+                                  borderRadius: BorderRadius.circular(12.0)),
+                              hintText: "Enter Email",
+                            ),
+                          ),
+                          const SizedBox(height: 25.0),
+                          const Text("Password",
+                              style: TextStyle(
+                                  fontSize: 18.0, fontWeight: FontWeight.w900)),
+                          const SizedBox(height: 15.0),
+                          TextFormField(
+                            controller: _passwordController,
+                            obscureText: true,
+                            textInputAction: TextInputAction.done,
+                            validator: (val) => val.length < 6
+                                ? 'Enter a password 6+ chars long'
+                                : null,
+                            decoration: InputDecoration(
+                              contentPadding: const EdgeInsets.only(left: 10.0),
+                              filled: true,
+                              fillColor: textField_fill_color,
+                              enabledBorder: OutlineInputBorder(
+                                  borderSide:
+                                      BorderSide(color: textField_fill_color),
+                                  borderRadius: BorderRadius.circular(12.0)),
+                              focusedBorder: OutlineInputBorder(
+                                  borderSide:
+                                      BorderSide(color: textField_fill_color),
+                                  borderRadius: BorderRadius.circular(12.0)),
+                              hintText: "Enter Password",
+                            ),
+                          ),
+                        ],
+                      ),
+                    ),
+                    const SizedBox(height: 35.0),
+                    // ignore: deprecated_member_use
+                    FlatButton(
+                      onPressed: () async {
+                        if (_formkey.currentState.validate()) {
+                          setState(() => load = true);
+                          dynamic result =
+                              await _auth.signInWithEmailAndPassword(
+                                  _emailController.text,
+                                  _passwordController.text);
+
+                          if (result == null) {
```

Comment:
Kindly add a else part , and navigate to the homescreen if authentication is successful